hs_model_program/static/css/model_program.css

```diff
@@ -0,0 +1,56 @@
+
+.mpElement{
+    font-weight: bold;
+    padding: 2px 2px 2px 2px;
+}
+
+a.mpElement::after {
+    display:block;
```

Comment:
When writing css, try to follow the formatting we are using. For example, use a space after colon.